magefile.go

```diff
@@ -1736,7 +1736,7 @@ func createTestRunner(matrix bool, singleTest string, goTestFlags string, batche
 	// Valid values are gcp-us-central1 (default), azure-eastus2
 	essRegion := os.Getenv("TEST_INTEG_AUTH_ESS_REGION")
 	if essRegion == "" {
-		essRegion = "gcp-us-central1"
+		essRegion = "aws-eu-central-1"
```

Comment:
We need to double check that changing cloud providers for deployment provisioning doesn't cause us any unexpected extra expenses.

CC @alexsapran

Reply:
I am not sure why you would expect extra expenses for GCP <-> AWS traffic as compared to GCP <-> Azure as we did when switching regions in QA...
Is there something specific to GCP <-> AWS data exchange ?

Reply:
Restored gcp region as default region. I will leave the aws support in anyways...